_includes/header.html

```diff
@@ -9,23 +9,28 @@
       <span class="note">{{ site.data.translations.note[page.lang] }}</span> {{ site.data.translations.development[page.lang] }}
     </div>
   {%- endif -%}
-  <div class="logobar">
-    <div class="logo">
-      <a href="{{ langSlug }}">
-        <img src="{{ relroot }}img/logo-RO.svg" alt="{{ site.data.translations.govlogo[page.lang] }}: {{ page.title }}">
+
+  <div class="header-yellow">
+    <div class="logobar">
+      <div class="logo">
+        <a href="{{ langSlug }}">
+          <img src="{{ relroot }}img/logo-RO.svg" alt="{{ site.data.translations.govlogo[page.lang] }}: {{ page.title }}">
+        </a>
+      </div>
+      <a href="{{ langSlug }}/faq" class="yellow__layout__faq-link">
+        {{ site.data.translations.faq[page.lang] }}
       </a>
+      {% include language-picker.html %}
     </div>
-  </div>
-  <div class="header-yellow">
     <div class="container">
-      <div class="header-yellow__text">
-        {% include language-picker.html %}
-        <h1>
-          {{ site.data.translations.header-title[page.lang] }}
-        </h1>
-        {% if site.data.translations.header-subtitle[page.lang] != undefined %}
-        <h2 class="subtitle">{{ site.data.translations.header-subtitle[page.lang] }}</h2>
-        {% endif %}
+      <div class="header-yellow__layout">
+        <img src="{{ relroot }}img/logo-app.svg" class="header-yellow__logo-badge"></img>
```

Comment:
Deze afbeelding heeft geen `alt`-attribuut. Dat zorgt ervoor dat in sommige screenreaders de URL van de afbeelding zal worden voorgelezen. Advies: voeg `alt=""` toe (want het icoon is decoratief tov de titel die er naast staat).

Reply:
+1